code/game/gamemodes/blob/powers.dm

```diff
@@ -16,24 +16,27 @@
 				continue
 			if(M.client)
 				to_chat(src, "<span class='warning'>There is someone too close to place your blob core!</span>")
-				return 0
+				return FALSE
 		for(var/mob/living/M in view(13, src))
 			if("blob" in M.faction)
 				continue
 			if(M.client)
 				to_chat(src, "<span class='warning'>Someone could see your blob core from here!</span>")
-				return 0
+				return FALSE
 		var/turf/T = get_turf(src)
 		if(T.density)
 			to_chat(src, "<span class='warning'>This spot is too dense to place a blob core on!</span>")
-			return 0
+			return FALSE
+		if(T.z != ZLEVEL_STATION)
```

Comment:
how does this check fix anything? it's not like the blob eye can move on a shuttle...

Reply:
you can still click on the shuttle tile

Reply:
which doesn't spawn your core there, so your ability to click the tile is irrelevant

Reply:
in any case the correct fix would be to keep the overmind from placing nodes on shuttles, which you haven't done